network-api/networkapi/wagtailpages/migrations/0026_auto_20180524_1225.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.13 on 2018-05-24 19:25
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+def mark_existing_petitions_as_legacy(apps, schema_editor):
+    petitions = apps.get_model("wagtailpages", "Petition")
+    for i in petitions.objects.all():
```

Comment:
I think you should be able to do `petition.objects.all().update(legacy_petition=True)` to have Django make the change in one SQL statement.

Reply:
handy!